tests/phpunit/CRM/Contribute/ActionMapping/ByTypeTest.php

```diff
@@ -260,7 +260,6 @@ public function testTokenRendering(): void {
       first name = {contact.first_name}
       receive_date = {contribution.receive_date}
       contribution status id = {contribution.contribution_status_id}
-      legacy style status = {contribution.status}
```

Comment:
@eileenmcnaughton I know you put this in here but is there anything that points conclusively that we are meant to be using name not label from this context (just figuring that the default install will have label = name for completed)

Reply:
@seamuslee001 yeah - good question - I've figured out it's coming from here 

https://github.com/civicrm/civicrm-core/blob/26219967a075d18e87fbfa84d4ce99e08de89e17/Civi/Token/TokenRow.php#L208-L212

(that function should be moved to Contribute_Tokens IMHO since it's not used from elsewhere & it's got some separation of concern issues)

So I just need to check what buildOptions would return there

Reply:
Aha -       'get' => "get: all options are returned, even if they are disabled; labels are translated.", - so it looks like it should be label

Reply:
Maybe worth altering the label in the test just to confirm that perhaps?

Reply:
hmm - you mean just as a way to test locally to double check my read of the code?

Reply:
Kind of / lock in that behaviour?

Reply:
@seamuslee001 I'm confused - because it's behaviour being removed so it can't be locked in?

Reply:
@eileenmcnaughton this is what I was thinking of https://github.com/civicrm/civicrm-core/commit/9fbcf5754e56234393aed03e9ce361e6a989c345, I would also suggest that in your upgrade script you change reference from :name to :label to match what was actually going on

Reply:
@seamuslee001 yeah we DO want to remove that token altogether though rather than lock it in. I can confirm locally that it IS label but I've already updated the PR for it to be label.

Reply:
@eileenmcnaughton I guess my point was more that at the present moment we aren't testing that it is the actual label to be returned and also not sure if we are testing anywhere that the translation works as well but I guess it felt like to me that we were swapping apples for oranges (going from testing that the label is outputted from {contribution.status} to using name {contribution.contribution_status_id:name})

Reply:
@seamuslee001 yeah I added label in - the follow up changes put financial_type_id through the same process & that one was testing label but I guess that one isn't an open PR yet since it was pending this one so I just added in label here